html/Task_4/style.css

```diff
@@ -0,0 +1,110 @@
+body {
+  width: 100%;
+  height: 100%;
+  box-sizing: border-box;
+  font-family: "Lato", sans-serif;
+}
+
+.container {
+  width: 80%;
+  margin: 0 auto;
+}
+
+.container .label {
```

Comment:
here and below you don't have to create bigger specify, you can just use .label, input[type="checkbox"] and so on